cmd/generate.go

```diff
@@ -50,12 +51,14 @@ func init() {
 	// complete a run.
 	generateCmd.Flags().StringVarP(&lifetime, "lifetime", "l",
 		"6h", "cluster lifetime")
+	generateCmd.Flags().StringVar(&usage, "usage", "cloud-report-2022", "the usage label for roachprod create. Default: cloud-report-2022")
```

Comment:
Not sure I follow how this label is intended to be used? 
Also, I'd suggest renaming since `usage` is typically reserved as the name of a function which displays usage information for command-line options, i.e., `--help`. How about renaming to a more generic label, e.g., `customer`?

Reply:
The label was discussed in dev-inf & cloud report channel and got adopted, I might remember wrong but think @jlinder or another colleague made the suggestion on the current name. I am good with either of them, except if we change it after we have existing labels for current runs including earlier test runs, then we need to track both labels to come up with the final cost, which looks like a big hassle.

Reply:
The `usage` label is specifically used to track cloud report cost starting from this year, other teams may use it with a different value to track a project's cost I guess too.

Reply:
Hi, 

@srosenberg `Usage` is a specific tag in our tagging taxonomy for cloud resources. The name fits well when used in a cloud resource's set of tags/labels, but, as you mentioned, it doesn't translate well to use in a CLI flag.

For this code, ideally, the name of the CLI flag here would be something other than `usage` since `usage` as a CLI argument has a strong, pre-existing meaning for CLI commands. Could we instead name the flag to be `usage-label` or `usage-tag`? 

Choosing `usage-tag` in the suggestion because it's shorter.

```suggestion
// The `Usage` tag/label identifies the use that a cloud resource is/was used for. This field 
// sets the value to use when starting resources using roachprod.
generateCmd.Flags().StringVar(&usage, "usage-tag", "cloud-report-2022", "the usage tag/label for roachprod create that identifies cloud report resources. Default: cloud-report-2022")
```

Reply:
> Choosing usage-tag in the suggestion because it's shorter.

Looks good for disambiguation.